Foundation/Thread.swift

```diff
@@ -270,6 +274,9 @@ open class Thread : NSObject {
     }
 
     open class var callStackSymbols: [String] {
+#if os(Android)
+            return []
```

Comment:
I think the return here should line up with the `return` of the `return backtraceAddresses` below. It's not rendering as that in the GitHub UI. Maybe a mix of tabs and spaces causing problems?

Reply:
Looks like all spaces here, but agree that `return` needs to be de-indented four spaces.